python/lsst/pipe/analysis/analysis.py

```diff
@@ -1018,6 +1025,84 @@ def plotQuiver(self, catalog, filename, log, cmap=plt.cm.Spectral, stats=None, d
         fig.savefig(filename, dpi=150)
         plt.close(fig)
 
+
+    def plotRhoStatistics(self, dataId, filename, log, enforcer, butler=None, camera=None, ccdList=None,
+                          tractInfo=None, patchList=None, hscRun=None, matchRadius=None, zpLabel=None,
+                          forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None):
+        fig, ax = plt.subplots(nrows=1, ncols=1, sharex=True)
+        axes = [ax]
```

Comment:
Why is this a single element list? It looks to only be used as axes[0] anyway.

Reply:
I was debating whether to have multiple panels or not, and this is the result of my confusion. It also makes changing the layout in the future easy, but I am resorting to multiple panels now as Morgan suggested